draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -43,75 +43,37 @@ organization="Ping Identity"
-Verifiers from obtaining claims irrelevant for the transaction at hand.
-SD-JWTs defined in this document enable such selective disclosure of JWT claims.
-
-One example of a multi-use JWT is an Issuer-signed
-credential that contains the claims about a subject, and whose authenticity can be
-cryptographically verified.
-
-Similar to the JWT specification on which it builds, this document is a product of the
-Web Authorization Protocol (OAuth) working group. However, while both JWT and SD-JWT
-have potential OAuth 2.0 applications, their utility and application is certainly not constrained to OAuth 2.0.
-JWT was developed as a general-purpose token format and has seen widespread usage in a
-variety of applications. SD-JWT is a selective disclosure mechanism for JWT and is
-similarly intended to be a general-purpose specification.
-
-While JWTs with claims describing natural persons are a common use case, the
-mechanisms defined in this document are also applicable to other use cases.
-
-In an SD-JWT, claims can be hidden, but cryptographically
-protected against undetected modification. When issuing the SD-JWT to
-the Holder, the Issuer includes the cleartext counterparts of all hidden
-claims, the so-called Disclosures, outside the signed part of the SD-JWT.
-
-The Holder decides which claims to disclose to a particular Verifier and includes the respective
-Disclosures in the SD-JWT to that Verifier. The Verifier
-has to verify that all disclosed claim values were part of the
-Issuer-signed JWT. The Verifier will not, however, learn any hidden claim
-values not disclosed in the Disclosures.
-
-This document also defines a format for SD-JWTs with Key Binding (SD-JWT+KB).
-By optionally sending an SD-JWT+KB to a
-Verifier, the Holder can prove to the Verifier that they hold the private key
-associated to the SD-JWT (i.e., using the `cnf` claim [@!RFC7800]). The strength of the binding is conditional upon the trust
-in the protection of the private key of the key pair an SD-JWT is bound to.
-
-SD-JWT can be used with any JSON-based representation of claims.
-
-This specification aims to be easy to implement and to leverage
-established and widely used data formats and cryptographic algorithms
-wherever possible.
+JSON-encoded data structures for exchange between systems are often secured against modification using JSON Web Signatures (JWS) [@!RFC7515].
+A popular application of JWS are JSON Web Tokens (JWT) [@!RFC7519], which are often used to represent a user's identity.
+An ID Token as defined in OpenID Connect [@?OpenID.Core], for example, is a JWT containing the user's claims created by the server for consumption by a client.
+In case the JWS is sent directly from the server to the client, as in OpenID Connect, the server can select which user claims to include in the JWT, minimizing the information shared with the client.
+
+A new model is emerging that decouples the issuance of a JWT from its presentation.
```

Comment:
the AS that issues the access token and the RS that consumes it are assumed to be the same entity though, right?

Reply:
Right. I was thinking in technical terms (server and client communication) while the decoupling happens on a different layer which actually is a new (trust) model. I believe, it would be good to focus more on that change of trust where the RS/Verifier is not a single trusted entity (and certainly not the same entity as the one issuing the JWT) but any third party.

Reply:
@ju-cu is not wrong here 
I've long made the complaint that the new "three-party model" sounds silly because we've had 3 party models for a long time.
But not sure how much of that can or should be captured or conveyed here.
And the previous intro might have kind of captured that but not particularly well IMO 
what about a bit o' qualification to this sentence?

```suggestion
A new model is emerging that more fully decouples the issuance of a JWT from its presentation.
```

Reply:
```suggestion
A new model is emerging that fully decouples the issuance of a JWT from its presentation.
```